streams/kr.m3u

```diff
@@ -211,3 +211,9 @@ http://220.95.92.160:9981/stream/channelid/1445999617
 http://210.210.155.37/dr9445/h/h20/index.m3u8
 #EXTINF:-1 tvg-id="" status="online",tvN Movies (Indonesian Subtitle) (576p)
 http://210.210.155.37/dr9445/h/h21/index.m3u8
+#EXTINF:-1 tvg-id="" status="online",JTBC(576p)
+https://tistory1.daumcdn.net/tistory/2864485/skin/images/CATV_211_53993BC5.m3u8
+#EXTINF:-1 tvg-id="" status="online",MBN(576p)
+http://hls-live.mbn.co.kr/mbn-on-air/600k/내가playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",CBS POWER RADIO (MUSIC4U)
+https://cms.zroad.tv/front/view?app_id=79&men_id=7134
```

Comment:
Goes to a webpage, if it's Audio-only you may propose it to https://github.com/LaneSh4d0w/IPRD

Reply:
Is https://aac.cbs.co.kr/cbs939/_definst_/cbs939.stream/playlist.m3u8 the stream you were looking for ? It's audio only if that's the case.